packages/babel-types/src/definitions/experimental.js

```diff
@@ -28,8 +29,20 @@ defineType("AwaitExpression", {
 defineType("BindExpression", {
   visitor: ["object", "callee"],
   aliases: ["Expression"],
+  // todo(ts): if this is breaking change - what node types should be here for old version?
   fields: !process.env.BABEL_TYPES_8_BREAKING
-    ? {}
+    ? {
+        object: {
+          validate: Object.assign(() => true, {
+            oneOfNodeTypes: ["Expression"],
+          }),
+        },
+        callee: {
+          validate: Object.assign(() => true, {
+            oneOfNodeTypes: ["Expression"],
+          }),
+        },
+      }
```

Comment:
This is a breaking change just because we don't validate those fields in Babel 7.

Reply:
so, for type definitions it is ok to keep it as `Expression`?

Reply:
Yeah